test/framework/kogitodataindex.go

```diff
@@ -73,18 +69,25 @@ func crInstallKogitoDataIndex(namespace string, replicas int32) error {
 
 func cliInstallKogitoDataIndex(namespace string, replicas int) error {
 	cmd := []string{"install", "data-index"}
-
-	// Get correct image tag
-	image := framework.ConvertImageTagToImage(infrastructure.DefaultDataIndexImageFullTag)
-	image.Tag = GetConfigServicesImageVersion()
-	cmd = append(cmd, "--image", framework.ConvertImageToImageTag(image))
-
+	cmd = append(cmd, "--image", framework.ConvertImageToImageTag(getDataIndexImage()))
 	cmd = append(cmd, "--replicas", strconv.Itoa(replicas))
 
 	_, err := ExecuteCliCommandInNamespace(namespace, cmd...)
 	return err
 }
 
+func getDataIndexImage() v1alpha1.Image {
+	if len(GetConfigDataIndexImageTag()) > 0 {
+		return framework.ConvertImageTagToImage(GetConfigDataIndexImageTag())
+	}
+
+	image := framework.ConvertImageTagToImage(infrastructure.DefaultDataIndexImageFullTag)
+	if len(GetConfigServicesImageVersion()) > 0 {
```

Comment:
There is always a default value for it (version.Version) so I guess we don't need that?

Reply:
updated